packages/frontend-shared/src/gql-components/topnav/UpdateCypressModal.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
Is this is the wrong directory? It's not using GraphQL.

Reply:
For me it belongs in the `topnav` directory with the other related components that make up the top nav, & the other factors driving the location of `topnav` itself don't matter too much. It is extracted into an SFC so that it's easier to read and test, but currently is only intended to be used as part of the top nav.